openpype/hosts/maya/api/pipeline.py

```diff
@@ -653,7 +653,7 @@ def on_task_changed():
 
     with lib.suspended_refresh():
         lib.set_context_settings()
-        lib.update_content_on_context_change()
+        lib.update_instances_frame_range()
```

Comment:
Should this also get the `lib.update_instances_asset_attribute()` call to behave as it did before?